src/PerimeterxDataEnrichment.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace Perimeterx;
+
+class PerimeterxDataEnrichment {
+
+public static function processDataEnrichment($pxCtx, $pxConfig) {
```

Comment:
a unit test for this part of the code would be nice

Reply:
will add